README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img align="center" width="250" src="golua.png"></p><br/>
```

Comment:
```suggestion
![Logo](golua.png)
```
Don't you think it's better to keep everything in Markdown? 
OBS.: Just a question! :smile: